sklearn/preprocessing/label.py

```diff
@@ -661,6 +668,20 @@ class MultiLabelBinarizer(BaseEstimator, TransformerMixin):
     >>> list(mlb.classes_)
     ['comedy', 'sci-fi', 'thriller']
 
+    Perform a one-hot encoding for y labels
```

Comment:
Relinking discussion [here](https://github.com/scikit-learn/scikit-learn/issues/5930#issuecomment-161134085). The `LabelBinarizer` doesn't necessarily do what we expect for 2-label problems. 

That being said, for 3-label problems, you'll get output like this:

```
>>> from sklearn.preprocessing import LabelBinarizer
>>> LabelBinarizer().fit_transform([1, 2, 3, 1])
array([[1, 0, 0],
       [0, 1, 0],
       [0, 0, 1],
       [1, 0, 0]])
```

I wasn't aware that the `LabelBinarizer` behaved differently for 2-label problems versus 3-label problems...